apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -23,19 +27,20 @@ const styles = {
  */
 const ScriptEditor = React.createClass({
   propTypes: {
-    beta: React.PropTypes.bool,
-    name: React.PropTypes.string.isRequired,
-    i18nData: React.PropTypes.object.isRequired,
-    hidden: React.PropTypes.bool,
-    loginRequired: React.PropTypes.bool,
-    hideableStages: React.PropTypes.bool,
-    studentDetailProgressView: React.PropTypes.bool,
-    professionalLearningCourse: React.PropTypes.bool,
-    peerReviewsRequired: React.PropTypes.number,
-    wrapupVideo: React.PropTypes.string,
-    excludeCsfColumnInLegend: React.PropTypes.bool,
-    projectWidgetVisible: React.PropTypes.bool,
-    projectWidgetTypes: React.PropTypes.arrayOf(React.PropTypes.string)
+    beta: PropTypes.bool,
+    name: PropTypes.string.isRequired,
+    i18nData: PropTypes.object.isRequired,
+    hidden: PropTypes.bool,
+    loginRequired: PropTypes.bool,
+    hideableStages: PropTypes.bool,
+    studentDetailProgressView: PropTypes.bool,
+    professionalLearningCourse: PropTypes.bool,
+    peerReviewsRequired: PropTypes.number,
+    wrapupVideo: PropTypes.string,
+    excludeCsfColumnInLegend: PropTypes.bool,
+    projectWidgetVisible: PropTypes.bool,
+    projectWidgetTypes: PropTypes.arrayOf(PropTypes.string),
+    teacherResources: PropTypes.arrayOf(resourceShape).isRequired,
```

Comment:
Most of this is just replacing React.PropTypes with PropTypes. `teacherResources` is new.